github-pr

```diff
@@ -12,11 +12,19 @@ def _check_required_fields(required, **args):
             sys.exit(1)
 
 
+def _return_status(commits):
+    commit_statuses = []
+    for commit in commits:
+        for commit_status in commit.get_statuses():
+            commit_statuses.append(commit_status.state)
+    return commit_statuses[-1]
```

Comment:
oh so the first one is the one want the status of? i was thinking if there were 
`(github_env)mbabineau@matt-bs-mbp:~/dev/github-pr$ github-pr list -r dataxu/jenkins-jobs --token=${GIT_TOKEN} -n 339
ID:Date 135223584:2014-12-01 23:38:20
Status: success
ID:Date 135271293:2014-12-02 02:00:03
Status: success
ID:Date 135274122:2014-12-02 02:06:46
Status: success
ID:Date 135274354:2014-12-02 02:07:17
Status: success
ID:Date 135315242:2014-12-02 04:01:52
Status: success
ID:Date 135276093:2014-12-02 02:11:51
Status: success
#339 [open]     master <- dev-warehouse-test-job            warehouse parallel test job - Commit/Build Status:success
Github Rate Limiting: 4964 remaining of max 5000`